tools: fix a usage of setting a region to tombstone (#1670)
pingcap/docs: dev/reference/tools/tikv-control.md

In a TiKV instance, you can use this command to set the status of some Regions to Tombstone. Then when you restart the instance, those Regions are skipped. Those Regions need to have enough healthy replicas in other TiKV instances to be able to continue writing and reading through the Raft mechanism.

Follow the two steps to set a Region to Tombstone:

> 1. Take off the corresponding Peer of this Region on the machine in `pd-ctl`:

Reviewer: Since `take off` has other meanings, how about using a clearer word?

```suggestion
1. Remove the corresponding Peer of this Region on the machine in `pd-ctl`:
```

    {{< copyable "shell-regular" >}}

    ```shell
    pd-ctl operator add remove-peer <region_id> <store_id>
    ```

2. Use the `tombstone` command to set a Region to Tombstone:

    {{< copyable "shell-regular" >}}

    ```shell
    tikv-ctl --db /path/to/tikv/db tombstone -p 127.0.0.1:2379 -r <region_id>
    ```

    ```
    success!
    ```

> **Note:**
>
> - The `tombstone` command only supports the local mode.

> > - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether PD can securely switch to Tombstone. Therefore, before setting a PD instance to Tombstone, you need to take off the corresponding Peer of this Region on the machine in `pd-ctl`.

Reviewer: Remove the content that has been changed to a step.

```suggestion
> - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether PD can safely switch to Tombstone.
```
